python/mim_control/robot_impedance_controller.py

```diff
@@ -52,6 +52,7 @@ def initialise_impedance_controllers(self, config_file):
                     int(
                         data_in["impedance_controllers"][ctrls]["start_column"]
                     ),
+                    data_in["impedance_controllers"][ctrls]["is_joint_active"],
```

Comment:
I don't find this entry in the robot_properties_solo. 

https://github.com/open-dynamic-robot-initiative/robot_properties_solo/blob/master/src/robot_properties_solo/robot_properties_solo/impedance_ctrl.yaml

Reply:
You are right. Should be added.